packages/react-dom/src/events/__tests__/TapEventPlugin-test.internal.js

```diff
@@ -29,6 +29,13 @@ let CHILD;
 
 let putListener;
 
+// TapEventPlugin requires the DOM fork of ResponderTopLevelEventTypes in order
+// to properly detect mouse events.
+// When building the production artifact, this is handled by our rollup config.
```

Comment:
See the first thought in my comment here: https://github.com/facebook/react/pull/12629#issuecomment-388506615